daprdocs/content/en/reference/components-reference/supported-middleware/_index.md

```diff
@@ -29,3 +29,4 @@ Table captions:
 | [Bearer]({{< ref middleware-bearer.md >}})                                     | Verifies a [Bearer Token](https://tools.ietf.org/html/rfc6750) using [OpenID Connect](https://openid.net/connect/) on a Web API | Alpha                      | v1|
 | [Open Policy Agent]({{< ref middleware-opa.md >}})                             | Applies [Rego/OPA Policies](https://www.openpolicyagent.org/) to incoming Dapr HTTP requests                                    | Alpha                      | v1|
 | [Uppercase]({{< ref middleware-uppercase.md >}})                               | Converts the body of the request to uppercase letters                                                                           | Stable(For local development) | v1|
+| [RouterChecker]({{< ref middleware-routerchecker.md >}})                       | Validates HTTP requests using regexp and filters out invalid requests                                                           | Alpha                      | v1|
```

Comment:
I notice that the Sentinel Middleware is missing from this list. This link https://docs.dapr.io/reference/components-reference/supported-middleware/middleware-sentinel/

Can you added this with a description?

| [Sentinel]({{< ref middleware-sentinel.md >}}) | Use Sentinel middleware to guarantee the reliability and resiliency of your application | Alpha | v1|